edge/test/integration/device/device_test.go

```diff
@@ -18,8 +18,11 @@ package device_test
 
 import (
 	"encoding/json"
+	"net/http"
+
```

Comment:
remove extra new line.

Reply:
Imports are separated as go, kubeedge and vendor imports, new line is added to make it obvious. Siddharth earlier told me to make this segregation.

Reply:
one line is enough to separate the imports. ? Is the segregation not clear with one new line ?

Reply:
Oh. Got it. Thanks.
The image above displays only one line, so I got confused.
Removing the extra new line.